modules/ossm-automatic-sidecar-injection.adoc

```diff
@@ -7,63 +7,76 @@
 [id="ossm-automatic-sidecar-injection_{context}"]
 = Enabling automatic sidecar injection
 
-When deploying an application, you must opt-in to injection by setting the `sidecar.istio.io/inject` annotation to `"true"`. Opting in ensures that the sidecar injection does not interfere with other {product-title} features such as builder pods used by numerous frameworks within the {product-title} ecosystem.
+When deploying an application, you must opt-in to injection by setting the `sidecar.istio.io/inject` annotation to `true` in the `deployment` object. Opting in ensures that the sidecar injection does not interfere with other {product-title} features such as builder pods used by numerous frameworks within the {product-title} ecosystem.
```

Comment:
The annotation should go to the pod, so, it should be in the spec.template section not in the "deployment" section:
![image](https://user-images.githubusercontent.com/1662329/158562799-0383804b-5b0e-4c0e-bb27-5127e38d255f.png)

Reply:
Lucas, I can't tell from the screen shot, is this a deployment.yaml file or a pod.yaml file?  It looks like it matches an object of `kind:Deployment` ,which is what I've got documented in this topic.

Reply:
It's a deployment. The template section will tell which annotations will go directly on the generated pods.

Reply:
Updating this to include the full path (`spec.template.metadata.annotations.sidecar.istio.io/inject`) for clarity.